hash/msx2_cart.xml

```diff
@@ -755,17 +755,17 @@ LZ93A13 (32 pin) - 8KB banks
             <feature name="mapper" value="M60002-0115FP" />
             <feature name="sram" value="D4464C-20L" />
             <dataarea name="rom" size="">
-                <rom name="jr_mate_1_v1.0" size="131072" crc="" sha1="" offset="0" status="nodump" />
-                <rom name="jr_mate_2_v1.0" size="131072" crc="" sha1="" offset="0x20000" status="nodump" />
+                <rom name="jr_mate_1_v1.0" size="131072" crc="" sha1="" offset="0" />
+                <rom name="jr_mate_2_v1.0" size="131072" crc="" sha1="" offset="0x20000" />
                 <!- Kanji rom ->
-                <rom name="tc531000p-6616" size="131072" crc="" sha1="" offset="0x40000" status="nodump" />
+                <rom name="tc531000p-6616" size="131072" crc="" sha1="" offset="0x40000" />
```

Comment:
Why did you remove the undumped status flag from these?

Reply:
> Why did you remove the undumped status flag from these?

Undone those changes.
Same situation as the previous post, I don't remember having made these changes.
But thanks for the warnings!